rootfs/etc/profile.d/Ω_finalize.sh

```diff
@@ -0,0 +1,17 @@
+## Perform any clean-up or post-setup actions
+
+# Set up command completion for command aliases
+
+_install_alias_completion
+
+# remove duplicates from PROMPT_COMMAND
```

Comment:
Help me understand... don't know why we would have duplicates

Reply:
We typically run at Shell Level 3: We start up, spawn a subshell with `assume-role`, then start another subshell with `chamber exec`. Every time we start these subshells, we rerun all the initialization, but `PROMPT_COMMAND` is inherited, so poorly behaved plugins (specifically `kube-ps1`) keep adding themselves to the `PROMPT_COMMAND` and run multiple times for every prompt. Or they would, if we did not do something about it.